iree/compiler/Dialect/VM/Target/C/test/global_ops.mlir

```diff
@@ -12,8 +12,11 @@ vm.module @global_ops {
   vm.global.i32 @c42 42 : i32
   vm.global.i32 @c107_mut mutable 107 : i32
 
+  // Skip forward declarations
+  // CHECK: DEFINE FUNCTIONS
+
   vm.export @test_global_load_i32
-  // CHECK-LABEL: iree_status_t global_ops_test_global_load_i32_impl(
+  // CHECK: iree_status_t global_ops_test_global_load_i32_impl(
```

Comment:
these should be static too, right? (or is that just not checked here?)

Reply:
Now the are